test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -120,106 +120,6 @@ def conduct_preprocessing_of_images_before_running_ecr_scans(image, ecr_client,
     return image
 
 
-@pytest.mark.usefixtures("sagemaker")
```

Comment:
removing this, no need in keeping anything that is marked as obselete

Reply:
Understood, this looks to be the old scanner.

Reply:
Ack